src/pyfaf/actions/pull_releases.py

```diff
@@ -83,6 +83,9 @@ def run(self, cmdline, db):
 
                 continue
 
+            if not opsys_plugin.eol and remote_status == "EOL":
```

Comment:
What about logging something in this case?

Reply:
debug is enough?

Reply:
Yes, sir.